### DISCUSSION TITLE
IBEX + Chromogenic Imaging?

### DISCUSSION THREAD

**Discussion by AdamWV** [IBEXImagingCommunity/ibex_imaging_knowledge_base#132] [Q&A] [answered]
Hi all, new IBEX user here. Has anyone tried to combine IBEX with chromogenic imaging? Would be very interested to hear of successes & struggles. My goal is to first perform IBEX and then use ACD's chromogen-based BaseScope detection method for RNA junctions. Thanks!

**Comment by colinchu72** [comment, marked as answer]
I've not done chromogenic, but Emily Speranza in the initial PNAS paper used Opal dyes, so you might be able to look at that. We did try a Beta-galactosidase kit on mouse retina sections after an IBEX run which is sort of chromogenic/enzymatic. It worked when done in the first, but not after 5 rounds of IBEX...

**Reply by AdamWV** [reply]
Hi Colin, thanks very much for your reply! We decided to change tack and are going to do a rolling circle amplification detection method instead, which will be fluorescent rather than chromogenic.

**Reply by radtkea** [reply]
Thanks Colin for helping Adam! 

Welcome to the IBEX community Adam! 

Glad you found an answer to your question and a path forward. When selecting fluorophores, please check out our [list](https://ibeximagingcommunity.github.io/ibex_imaging_knowledge_base/fluorescent_probes.html) of tested probes. 

We actually found several of the Atto dyes did not dye inactivate, right @colinchu72? I see they are not listed on the site, yet. Shame on us:)

Best of luck. Reach out if you have any further questions.
A

**Reply by AdamWV** [reply]
Thank you both very much for your helpful replies! I have a follow-up question. For the rolling circle amplification, I need oligo-fluorochrome detection sequences. In the PNAS IBEX paper supplementary table 5 there are several imaging sequences presented, which would be ideal for my needs. My question is: Did any of these sequences perform better than the others? Among them, is there a "top sequence" you would recommend that gave the best signal:noise?

**Reply by colinchu72** [reply]
The oligos we used in Supplementary Table 5 all worked - it's hard to say if one worked better than another though due to the better antibody, target or the sequence of the oligo though. I think if you use any of them they should work, choose the brighter fluorophores (AF488, AF647, ATTO550).

If you are doing rolling circle amplification, we have had success with the approach used in the CoppaFISH paper protocol (https://www.nature.com/articles/s41586-022-04915-7).

**Reply by radtkea** [reply]
Hello again:)

The sequences were complementary to the Total-Seq antibodies Colin used. I don’t think he designed or optimized them. We never got the B220 or CD11c antibodies to work well. You will note that they are non-IBEX antibodies. B220 was a bit of a surprise because that is such a great clone that works in many direct conjugates. It yielded a nuclear label with the oligo-conjugated antibodies. 

This might be a helpful resource for you: https://paintshop.io/
I know people in the lab and can connect you, if you have questions.

Best of luck!
A
